docs/conf.py

```diff
@@ -323,7 +323,8 @@ def source_replace(app, docname, source):
 source_replacements = {
     "{PLONE_BACKEND_MINOR_VERSION}": "6.0",
     "{PLONE_BACKEND_PATCH_VERSION}": "6.0.8",
-    "{NVM_VERSION}": "0.39.3",
+    "{NVM_VERSION}": "0.39.5",
+    "{SUPPORTED_PYTHON_VERSIONS}": "3.8, 3.9, 3.10, 3.11",
```

Comment:
I realized that the grammar gets tricky. Sometimes we might say "or" such as when the developer may choose a single Python version to run on.

- https://github.com/plone/documentation/blame/bff9287a3af7c6d64ef73d604ff2a4d82834d4de/docs/contributing/documentation/setup-build.md#L28-L28
- https://github.com/plone/documentation/blame/bff9287a3af7c6d64ef73d604ff2a4d82834d4de/docs/classic-ui/theming/barceloneta.md#L25-L25 

Other times we might say "and" when the package supports all the given Python versions.

- https://github.com/plone/documentation/blame/bff9287a3af7c6d64ef73d604ff2a4d82834d4de/docs/backend/upgrading/version-specific-migration/upgrade-to-60.md#L187-L187

Fortunately the latter form is used only in documentation that should never update again.

```suggestion
    "{SUPPORTED_PYTHON_VERSIONS}": "3.8, 3.9, 3.10, or 3.11",
```

It would be good to update those two other "or" examples, either in this PR or a new one after this is merged.